src/Poll.ts

```diff
@@ -162,8 +169,6 @@ export class Poll {
     private buildVoteTally(overrideAnon: boolean, votes: any, key: string): SectionBlock | null {
         const users: string[] = votes[key].split(",");
         users.splice(0, 1);
-        // Don"t bother with empty votes
-        if (users.length === 0) return null;
```

Comment:
Does this mean we're now going to show an option if it has 0 votes?

Reply:
Originally, yes. However since there appears to be a want for anonymous polling, we can re-add it.

Reply:
Well even with non anonymous polling we don't want to show votes with 0 people in the results block.

Reply:
We need to though, otherwise how will we know what the vote options are?

Reply:
But it looks like you kept the buttons but disabled voting.

Reply:
There were previous iterations (see previous commits) where the buttons were deleted. However I re-added the buttons since they store information in regards to who voted. Since we kept the buttons, this will be re-added.